app/src/main/kotlin/com/google/samples/apps/nowinandroid/MainActivity.kt

```diff
@@ -126,17 +133,25 @@ class MainActivity : ComponentActivity() {
                 onDispose {}
             }
 
-            CompositionLocalProvider(LocalAnalyticsHelper provides analyticsHelper) {
+            val appState = rememberNiaAppState(
+                windowSizeClass = calculateWindowSizeClass(this),
+                networkMonitor = networkMonitor,
+                userNewsResourceRepository = userNewsResourceRepository,
+                timeZoneMonitor = timeZoneMonitor,
+            )
+
+            val currentTimeZone by appState.currentTimeZone.collectAsState()
```

Comment:
Should this be `collectAsStateWithLifecycle`, to unregister the broadcast receiver if the app is backgrounded?

Reply:
Yeah, sounds good.